### PR TITLE
Enable Google Tag manager

### PR DESCRIPTION
**Pull request by kronnox**
Part of #365 

Test URLs:
- Before: 
  - https://main--shredit--stericycle.aem.page/en-us
- After: 
  - https://martech-changes--shredit--stericycle.aem.page/en-us

### CODE REVIEW

File: scripts/martech.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  window.digitalData = {
+    page: {
+      pageInfo: {
+        brand: 'sid',
+        pageName: getMetadata('og:title'),
+        destinationURL: window.location.href,
+        referringURL: document.referrer,
+        language: getLocaleAsBCP47(),
+        pageURL: window.location.href,
+        pageType: 'homepage', // TODO: parse pagetype dynamically?
```
Comment (ericvangeem) [inline scripts/martech.js:15]: The previous implementation used the template name for `pageType`. For example [this location page](https://www.shredit.com/en-us/service-locations/omaha) on the prod site has a meta tag `<meta name="template" content="location"/>`

File: scripts/martech.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        pageDescription: getMetadata('og:description'),
+        country: 'US', // TODO: fetch country dynamically?
```
Comment (ericvangeem) [inline scripts/martech.js:17]: We can just grab the country code from the URL path, e.g. /en-**us**/

**Review by ericvangeem — CHANGES_REQUESTED** [review]

File: scripts/martech.js (hunk 1 above)
Comment (kronnox) [reply scripts/martech.js:15]: Yes, indeed. But it won't be a 1:1 match. I don't know yet what the pageType property is used for in you rules. Depending on that the results may vary. But I'm happy to use the template metadata as a replacement for now; just expecting for things to break.

File: scripts/martech.js (hunk 2 above)
Comment (kronnox) [reply scripts/martech.js:17]: Yes, but this still needs a custom conversion to the correct country code format. We can add that later as we will only serve the us site for now. But shouldn't be a huge task to do so anyway. Just wanted to get these changes out as quickly as possible so testing can progress

**Comment by kronnox** [comment]
@ericvangeem Added the additional logic for `dataLayer` event processing

**Review by ericvangeem — APPROVED** [review]

**Pull request merged**